config/features/p2lb/field.field.node.page.field_v3_conversion_revision_id.yml

```diff
@@ -13,7 +13,9 @@ label: 'Conversion revision ID'
 description: ''
 required: false
 translatable: false
-default_value: {  }
+default_value:
+  -
+    value: '1'
```

Comment:
I don't think we need this. I think we just need to hook into the entity save process and check if the node is being created for the first time and if it is store the newly created revision ID as the value.

Reply:
This seemed like a smaller footprint than custom code

Reply:
node_presave doesn't have a revision id yet. entity_insert has the revision id but modifying the p2lb revision field would count as another save. we recently ran into this with ITS alerts with needing a temp field value and the nid at the same time.

by using the same dummy revision id it also allows us to track and identify which nodes started as v3 while p2lb is enabled. this data will be deleted as part of cleanup.